src/amazonsqs/amazonsqs_connector.bal

```diff
@@ -136,10 +135,10 @@ public type Client client object {
     # + attributes - Non-mandatory parameters for receiving a message
     # + return - If success, details of the received message, else returns error
     public remote function receiveMessage(string queueResourcePath, map<string> attributes) 
-        returns @tainted InboundMessage[]|OperationError {
-        string amzTarget = "AmazonSQSv20121105.ReceiveMessage";
+        returns @tainted InboundMessage[]|ErrorOperation {
+        string amzTarget = AMAZON_SQS_API_VERSION + "." + ACTION_RECEIVE_MESSAGE;
         string payload =  "";
-        payload = payload + "&Action=ReceiveMessage";
+        payload = payload + "&Action=" + ACTION_RECEIVE_MESSAGE;
```

Comment:
What about `&Action=` can we make this a constant?

Reply:
Added a new constant to string _"Action"_, and passed to newly created method.